src/main.tf

```diff
@@ -108,6 +108,18 @@ locals {
         maintainers = ["services", "drone-engineering"]
       }
     }
+    "dao-gbc" = {
+      default_branch = "main"
+      description    = "Arrowair Grants and Bounties Committee"
+      visibility     = "public"
+      owner_team     = "services"
```

Comment:
We might want to add a new group actually, to make it the owner. We do have the 'documentation' group which I've added which might be valid for this. But it could also be that we need an 'operations' or 'core' group for this.
Members of the owner group are the only ones that can approve PRs

Reply:
Agreed! Maybe even 2. A growth-and-ops group and governance group perhaps?

Reply:
The 2 new groups have been created, so you can update it to use those instead